openc3/lib/openc3/core_ext/string.rb

```diff
@@ -389,4 +389,17 @@ def to_utf8!
     end
   end
 
-end # class String
+  def comment_erb
+    # Split with -1 to avoid stripping empty strings
+    output = self.split("\n", -1).collect! do |line|
+      # If we have a commented out line (starts with #)
+      # which contains an ERB statement (<% ...)
+      # then comment out the ERB statement (<%# ...)
+      if line.strip() =~ /^#.*<%/
```

Comment:
Should allow whitespace before the # sign.

Reply:
I'm explicitly not allowing trailing ERB. See comment. Also I added the ability to disable ERB commenting with `#%`